phpBB/styles/prosilver/theme/bidi.css

```diff
@@ -502,10 +502,18 @@ li.breadcrumbs span:first-child > a {
 
 .rtl blockquote cite {
 	/* Username/source of quoter */
-	margin-right: 20px;
+	margin-right: 0;
 	margin-left: 0;
 }
 
+.rtl blockquote cite:before {
+	/* Font Awesome quote-right */
+	content: '\f10e';
+	font-family: FontAwesome;
```

Comment:
The base element specifies quote-left, for the RTL element we need quote-right, so I'm not clear on what's redundant here.